modules/mtl/arrow-mtl-data/src/test/kotlin/arrow/mtl/KleisliTest.kt

```diff
@@ -58,6 +60,9 @@ class KleisliTest : UnitSpec() {
 
   init {
 
+    val cf: (Int) -> Kleisli<Kind<ForConst, Int>, Int, Int> = { Kleisli { it.const() } }
+    val g = Gen.int().map(cf) as Gen<Kind<KleisliPartialOf<ConstPartialOf<Int>, Int>, Int>>
```

Comment:
This can also be done with `GenK<KleisliPartialOf<F, R>>` by taking a `GenK<F>` and a `Gen<R>`

Reply:
ok